sharing/README.md

```diff
@@ -25,3 +25,172 @@ High level features include:
+    | [2021-03-01,2021-12-01] |
+
+## Example Scenarios
+
+In this section we will be working with some data, providing an example scenario for a rule and showing what the rule looks like
+
+The data we will be working with has two tables from the ODM, **Sample** and **Site**. It does not include all the columns present in these tables. The rows in the Sample and Site table respectively are shown below:
+
+| sampleID | siteID | dateTime   | type    | sizeL | notes  | 
+|----------|--------|------------|---------|-------|--------|
+| 1        | 1      | 2021-08-19 | rawWW   | 5     | Note 1 |
+| 2        | 1      | 2021-08-18 | sweSed  | 3     | Note 2 |
+| 3        | 2      | 2021-08-17 | pstGrit | 15    | Note 3 |
+| 4        | 2      | 2020-01-10 | water   | 45    | Note 4 |
+
+| siteID | name                 | publicHealthDepartment | type   |
+|--------|----------------------|------------------------|--------|
+| 1      | University of Ottawa | Ottawa Public Health   | school |
+| 1      | University of Laval  | Laval Public Health    | school |
+
+
+1. Remove rows whose site ID in the Sample table is 2 for Ottawa Public Health (OPH)
+
+    | sharedWith | tableName | variableName | direction | filterValue |
+    |------------|-----------|--------------|-----------|-------------|
+    | OPH        | Sample    | siteID       | row       | 1           |
+
+2. Remove rows from the Sample table whose type is `rawWW` or `sweSed` for the Public Health Agency of Canada (PHAC)
+
+    | sharedWith | tableName | variableName | direction | filterValue  |
+    |------------|-----------|--------------|-----------|--------------|
+    | PHAC       | Sample    | type         | row       | rawWW;sweSed |
+
+3. Remove the type column from all tables for Laval Public Health (LPH)
+
+    | sharedWith | tableName | variableName | direction | filterValue  |
+    |------------|-----------|--------------|-----------|--------------|
+    | LPH        | all       | type         | column    | all          |
+
+4. Remove samples taken in the year 2021 and those whose volume is less than 5 for OPH and PHAC
+
+    | sharedWith | tableName | variableName | direction | filterValue             |
+    |------------|-----------|--------------|-----------|-------------------------|
+    | OPH;PHAC   | Sample    | dateTime     | row       | [2021-01-01,2021-12-01] |
+    | OPH;PHAC   | Sample    | sizeL        | row       | [Inf,5]                 |
+
+5. Remove the notes column from the Sample table and rows from the Site table that belong to the University of Ottawa for LPH
+
+    | sharedWith | tableName | variableName           | direction | filterValue             |
+    |------------|-----------|------------------------|-----------|-------------------------|
+    | LPH        | Site      | notes                  | column    | all                     |
+    | LPH        | Sample    | publicHealthDepartment | row       | Ottawa Public Health    |
+
+## Sharing CSV Columns
```

Comment:
The names in this section are different from the `sharing_data.csv` file.

Maybe have this section earlier?

Reply:
The `shared_data.csv` file is old. I'll delete it.